ui/ducks/send/send.js

```diff
@@ -570,7 +570,7 @@ const slice = createSlice({
       state.recipient.nickname = action.payload.nickname;
       state.draftTransaction.id = action.payload.id;
       state.draftTransaction.txParams.from = action.payload.from;
-      slice.caseReducers.updateDraftTransaction(state);
+      slice.caseReducers.validateSendState(state);
```

Comment:
@brad-decker I banged my head against the wall here for a while and found that this is required for updating the validating and updating the state.status after modifying the state.stage here when navigating back to edit in a send-token flow. Let me know what you think?